res/TensorFlowLiteRecipes/BroadcastTo_001/test.recipe

```diff
@@ -0,0 +1,30 @@
+# BroadcastTo Op supports op version 2 and 3.
+# BroadcastTo of Float type is version 2 and Quantized BroadcastTo is version 3.
+# We are using tflite kernels for float values. so, the version should be 2.
+# https://github.com/tensorflow/tensorflow/blob/ce2d218fc88c20b1d81abb017eabd436779865e4/tensorflow/lite/kernels/register.cc#L306-L311
```

Comment:
I think this can be removed. this line is too long and 46224 explains about the version.